server/app/routes/users.router.js

```diff
@@ -8,8 +8,17 @@ var Game = db.Game;
 var Task = db.Task;
 var Event = db.Event;
 
+
+//For use in autocomplete to invite players
+router.get('/invite', function(req,res, next){
```

Comment:
This url may be a little misleading / non-standard. It seems like the resource being retrieved would be an `invite`, but really it's an array of users. Maybe you could roll this logic into the `/api/users` route somehow? If so you might want to like embed the `$like` in the query params directly, so your query could just look like: `{where: req.query}`.